src/libexpr/print.cc

```diff
@@ -41,44 +43,49 @@ bool isReservedKeyword(const std::string_view str)
     return reservedKeywords.contains(str);
 }
 
+// Returns 'true' if the character is a symbol that can't be used in a variable name.
+bool isSymbolProhibited(const char& symbol) {
+    bool symbolAllowed = (std::isalnum(symbol) || symbol == '_' || symbol == '-'|| symbol == '\'');
```

Comment:
> Checks if the given character is an alphanumeric character as classified by the current C locale.

([cppreference.com](https://en.cppreference.com/w/cpp/string/byte/isalnum))

I don't think we want to get the environment involved.
I'd be happy to use something that's always ASCII based, but this is not.
Also a potential performance hazard, although maybe the library is clever enough to tabulate it only once or something.

Reply:
Okay, I will think about it